parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -79,6 +79,12 @@ impl Into<Token> for Message {
 	}
 }
 
+impl From<u32> for AggregateMessageOrigin {
+	fn from(value: u32) -> Self {
+		AggregateMessageOrigin::Parachain(value.into())
+	}
+}
+
```

Comment:
Unrelated to this PR, but this breaks when building Cumulus when enabling the benchmarks feature.